FoliCon/Modules/utils/IconUtils.cs

```diff
@@ -86,7 +87,15 @@ public static class IconUtils
         ratingVisibility = string.IsNullOrEmpty(rating) ? "Hidden" : ratingVisibility;
         if (!string.IsNullOrEmpty(rating) && rating != "10")
         {
-            rating = !rating.Contains('.') ? $"{rating}.0" : rating;
+            if (double.TryParse(rating, out var ratingValue))
```

Comment:
as we have already did the formatting at ListItem, no need to do here as the data would have already been formatted.